### DISCUSSION TITLE
SelectGroup

### DISCUSSION THREAD

**Discussion by jiblett1000** [chakra-ui/ark#843] [Q&A] [answered]
Hey,

Just wondering the best way to use `SelectOptionGroup` and `SelectOptionGroupLabel`. It's not currently included in "story" within the vue select directory.

**Comment by anubra266** [comment, marked as answer]
@jiblett1000 https://arkxamples.vercel.app/select?framework=VUE

**Reply by jiblett1000** [reply]
Thanks 😊
